serverless.yml

```diff
@@ -27,23 +31,41 @@ functions:
   mttr:
     handler: src/mttr.handler
     events:
-      - sns:
-          arn: ${self:custom.topicArn}
+      - cloudwatchEvent:
+          name: mttr-ops-metrics-alarm-state-change
+          description: ops health monitoring
+          enabled: true
+          event:
+            source:
+              - "aws.cloudwatch"
+            detail-type:
+              - "CloudWatch Alarm State Change"
   mttf:
     handler: src/mttf.handler
     events:
-      - sns:
-          arn: ${self:custom.topicArn}
+      - cloudwatchEvent:
+          name: mttf-ops-metrics-alarm-state-change
+          enabled: true
+          event:
+            source:
+              - "aws.cloudwatch"
+            detail-type:
+              - "CloudWatch Alarm State Change"
   mtbf:
     handler: src/mtbf.handler
     events:
-      - sns:
-          arn: ${self:custom.topicArn}
-
+      - cloudwatchEvent:
+          name: mtbf-ops-metrics-alarm-state-change
+          enabled: true
+          event:
+            source:
+              - "aws.cloudwatch"
+            detail-type:
+              - "CloudWatch Alarm State Change"
 
 resources:
   Resources:
     MetricTopic:
```

Comment:
Is this topic still required?

Reply:
I think it's required because it's used in example alarm but it can be moved to the other serverless.yml

Reply:
See comment above - this can probably be removed.

Reply:
done